tdrs-backend/tdpservice/parsers/schema_defs/ssp/m1.py

```diff
@@ -133,15 +133,20 @@
                     required=True,
                     validators=[validators.notEmpty()]
                 ),
-                Field(
+                TransformField(
+                    zero_pad(3),
                     item="2",
-                    name='COUNTY_FIPS_CODE',
+                    name="COUNTY_FIPS_CODE",
                     friendly_name="county fips code",
-                    type='string',
+                    type="string",
                     startIndex=19,
                     endIndex=22,
                     required=True,
-                    validators=[validators.isNumber(),]
+                    validators=[
+                        validators.isInStringRange(1, 999),
+                        validators.fieldHasLength(3),
+                        validators.isNumber()
```

Comment:
are all three of these necessary?

Reply:
I think the length check is redundant. Verifying with Alex about the range validator.

Reply:
Talked with Alex and the only validation here is that it needs to be a number and have leading zeros if it doesnt already.